<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Workbench</class>
 <widget class="QMainWindow" name="Workbench">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The OpenCV Workbench</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Camera"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad_Config"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Video</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionRecord"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <addaction name="actionFace_Detect"/>
    <addaction name="actionScuba_Face_Label"/>
    <addaction name="actionVideo_Object_Label"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuVideo"/>
   <addaction name="menuProcess"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionOpen_Camera">
   <property name="text">
    <string>Open Camera</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionFace_Detect">
   <property name="text">
    <string>Face Detect</string>
   </property>
  </action>
  <action name="actionLoad_Config">
   <property name="text">
    <string>Load Config</string>
   </property>
  </action>
  <action name="actionScuba_Face_Label">
   <property name="text">
    <string>Scuba Face Label</string>
   </property>
  </action>
  <action name="actionVideo_Object_Label">
   <property name="text">
    <string>Video Object Label</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Workbench.qrc"/>
 </resources>
 <connections/>
</ui>
